doc/operator-guide/schema-migration-process.rst

```diff
@@ -2,8 +2,108 @@
 Schema Migration Process
 ########################
 
-* Add columns to sdm_schemas
-* Create alembic migration
-* Test migration and code to populate the new columns/tables at TTS/BTS if Summit schema is changing
+* How to add columns to sdm_schemas
+
+  - Edit the repository at https://sdm-schemas.lsst.io/ to apply your schema changes to any of the ``cdb_*.yml`` schemas.
+  - Check that valid SQL tables can be created with ``felis`` (https://felis.lsst.io/user-guide/cli.html#felis-validate) using:
+
+  .. code-block:: bash
+
+    pip install lsst-felis
+
+    felis validate [options] [edited cdb_*.yml files]
+
+  - Create a Jira ticket and assign it to Valerie to create a schema migration. (Is there a separate migration request process?)
+
+
+* Create an Alembic Migration
+
+  - Alembic (https://alembic.sqlalchemy.org/en/latest/front.html) keeps track of versioning by autogenerated migrations to sync the test stands and summit databases.
+  - Versioning our database schema changes allows us to apply edits and move the database’s state forward or backward as needed.
+
+  - How to create an Alembic migration to deploy your new ``sdm_schema`` edits:
+
+    1. Create an Alembic migration on your ``consdb`` ticket branch.
+    2. Use the script ``consdb/alembic-autogenerate.py`` to generate Alembic migrations. You’ll need the following environment variables:
+
+       - ``SDM_SCHEMAS_DIR`` - Points to a local clone of ``sdm_schemas`` with your ``cdb_*`` schema changes.
+
+    3. Run ``alembic-autogenerate`` to create version files in respective database-named directories in ``consdb``.
+
+       - Follow the directions in the header of the script.
+
+    4. Manually edit the generated files in ``consdb/alembic/<table-name>/`` to:
+
+       - Remove the ``visit1`` and ``ccdvisit1`` views.
```

Comment:
We should modify the migration script so that it strips the views automatically if they always show up erroneously. It is too error prone to expect them to be taken out every time by hand.

Reply:
Yes, I think we have covered this since this comment was made.